tests/unittests/OperatorTest.cpp

```diff
@@ -4926,6 +4926,58 @@ TEST_P(OperatorTest, FP16Max) {
   }
 }
 
+/// Helper to test Broadcast Max/Min using \p DTy and \p NTy
+template <typename DataType, typename NodeType>
+static void testBroadcastMaxMin(glow::PlaceholderBindings &bindings,
+                                glow::Module &mod, glow::Function *F,
+                                glow::ExecutionEngine &EE, ElemKind DTy) {
+
+  PseudoRNG PRNG;
```

Comment:
nit: We always use the module's PRNG for initializing, not a local one. I.e. `mod_.getPRNG()`.

Reply:
Done.